File: problem/part-1/solutions/LongestWordFinder.java
public class LongestWordFinder {
  public static void main(String[] args) {
    String mySentence = "Hello, I am a sentence with many different word lengths.";
      System.out.println(longestWord(mySentence));
  }
      
  public static String longestWord(String sentence) {
    String[] words = sentence.split(" ");
    String longestSoFar = "";
    for (int i = 0; i < words.length; i++) {
      if (words[i].length() > longestSoFar.length()) {
        longestSoFar = words[i];
      }
    }
    return longestSoFar;
  }
}

File: problem/part-1/solutions/LongestWordFinder.php
<?php

$sentence = "This is a long sentence with words of multiple lengths for inputting into your function.";

$words = explode(" ", $sentence);

$longest_word = "";

foreach($words as $word) {
    if (strlen($word) > strlen($longest_word)) {
        $longest_word = $word;
    }
}

print $longest_word . "\n";

